src/arch/arm/config.cmake

```diff
@@ -188,7 +194,11 @@ if(
     OR KernelArmCortexA35
     OR KernelArmCortexA53
     OR KernelArmCortexA57
+    OR KernelArmCortexA72
 )
+    # According to https://developer.arm.com/documentation/100095/0001/functional-description/about-the-cortex-a72-processor-functions/components-of-the-processor
```

Comment:
ditto?

Reply:
I'd prefer to keep the URL in the source, as this is what most people in the future will be looking at primarily.

Reply:
I agree with this one.

Reply:
I think this comment should be in a place more specific to the a72 processor? This comment is in a block that applies to 6 different processor types, but is only specific to the a72 which reduces its relevance.  Arguably, it's getting to the point where having micro-arch specific config spread around many config files makes it hard to tell if everything is correctly added for a new micro-arch.

Reply:
Let's postpone this as a refactoring step after merging this.